docs/reference/mapping/params/ignore-above.asciidoc

```diff
@@ -2,6 +2,9 @@
 === `ignore_above`
 
 Strings longer than the `ignore_above` setting will not be indexed or stored.
+For arrays of strings, `ignore_above` will be applied for each array element and string elements longer than `ignore_above` will not be indexed or stored.
```

Comment:
minor: maybe `...each array element separately and...` would be clearer here?

Reply:
Will fix, thanks!

Reply:
Fixed in https://github.com/elastic/elasticsearch/pull/33057/commits/eac5614eef0c4b527a94ad19f93103134848da0d as per your suggestion, I think it looks clearer now WDYT?